tcf_website/templates/common/rating_card.html

```diff
@@ -40,7 +40,7 @@ <h5>{{ subtitle }}</h5>
                   <i class="fas fa-chart-bar"></i>&nbsp;GPA
                 </small>
                 <p class="mb-0 info" id="gpa">
-                  {% if gpa %}
+                  {% if gpa and gpa <= 4 and gpa >= 0%}
```

Comment:
minor nitpick: should have a space between 0 and %

Reply:
Done